libs/platform/src/lib/table/table.service.ts

```diff
@@ -1,6 +1,7 @@
 import { EventEmitter, Injectable } from '@angular/core';
 import { BehaviorSubject, Subject } from 'rxjs';
 import { skip } from 'rxjs/operators';
+import equal from 'fast-deep-equal';
```

Comment:
I see you added import from a third-party lib, but it's not reflected as a dependency in package.json file.
You might also want to add it as a dependency for the @fundamental-ngx/platform/table submodule

Reply:
@N1XUS sorry, couldn't react to your comment before this was merged. This library is used internally by angular, so it's always included. I'll add it as a dependency to package.json

Reply:
@N1XUS https://github.com/SAP/fundamental-ngx/pull/7105